optima/results.py

```diff
@@ -470,10 +470,10 @@ def assemble(key):
             self.other['only'+healthkey].tot =  process(allpeople[:,healthinds,:,:][:,:,:,indices].sum(axis=(1,2))) # Axis 1 is populations
 
         return None
-        
-        
-    def export(self, filename=None, folder=None, bypop=True, sep=',', ind=None, key=None, sigfigs=3, writetofile=True, asexcel=True, verbose=2):
-        ''' Method for exporting results to an Excel or CSV file '''
+
+    def export(self, filename=None, folder=None, bypop=True, sep=',', ind=None, key=None, sigfigs=3, writetofile=True,
+               comparisontab=False, asexcel=True, verbose=2):
```

Comment:
I think this logic should be rethought a bit...I don't see any reason to _not_ always include the comparison tab in addition to the other sheets for a Multiresultset but I also don't see why you would want it for a single Resultset, so my preference would be to change around the logic a bit and have comparisontab only apply to `Multiresultset` and just be true (not a kwarg).

Reply:
@cliffckerr My hack only works for optimization results (baseline vs optimized), could you advice on the best way to make it the default behavior only whenever it is an optimization multiresult?